packages/components/package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "thunderstore-components",
+  "name": "@thunderstore/components",
   "version": "0.1.0",
   "description": "Shared components for Thunderstore",
-  "repository": "https://github.com/thunderstore-io/thunderstore-ui/tree/master/thunderstore-components",
-  "main": "./dist/index.js",
+  "repository": "https://github.com/thunderstore-io/thunderstore-ui/tree/master/packages/components",
```

Comment:
From the [docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository):
> The URL should be a publicly available (perhaps read-only) url that can be handed directly to a VCS program without any modification. It should not be a url to an html project page that you put in your browser. It's for computers.

A relevant example:
```json
{
  "repository": {
    "type": "git",
    "url": "https://github.com/facebook/react.git",
    "directory": "packages/react-dom"
  }
}
```